src/PlotlyKaleido.jl

```diff
@@ -20,6 +20,15 @@ const P = Pipes()
 const _mathjax_url_path = "https://cdnjs.cloudflare.com/ajax/libs/mathjax"
 const _mathjax_last_version = v"2.7.9"
 
+const _error_msg = Ref("")
```

Comment:
I don't really like introducing another global mutable for this. I mean, in its current form is not threadsafe either way, but I think making this part of the `Pipe` type to keep things in one place wouldn't hurt.

Reply:
I moved this error storage string inside the `Pipes` type.
Was that what you were suggesting @BeastyBlacksmith?